src/test/shell/integration/cpp_test.sh

```diff
@@ -90,4 +90,30 @@ EOF
   bazel build //a || fail "build failled"
 }
 
+function test_no_recompile_on_shutdown() {
+  mkdir -p a
+  cat > a/BUILD <<EOF
+cc_binary(name="a", srcs=["a.cc"], deps=["b"])
+cc_inc_library(name="b", hdrs=["b.h"])
+EOF
+
+  cat > a/a.cc <<EOF
+#include <a/b.h>
+
+int main(void) {
+  return B_RETURN_VALUE;
+}
+EOF
+
+  cat > a/b.h <<EOF
+#define B_RETURN_VALUE 31
+EOF
+
+  bazel build -s //a >& $TEST_log || fail "build failed"
+  expect_log "Compiling a/a.cc"
```

Comment:
This is the excerpt from the log that I'm looking to match (assuming the info line doesn't appear) which is always present with -s for a recompilation:

`>>>>>>>>> # //a:a [action 'Compiling a/a.cc']`

Do you want to match something else?